libnavigation-base/src/main/java/com/mapbox/navigation/base/route/model/PointNavigation.kt

```diff
@@ -0,0 +1,6 @@
+package com.mapbox.navigation.base.route.model
+
+data class PointNavigation(
```

Comment:
we also have android framework class `Location` that pretty much represents the same...

Reply:
discussed, restored